molgenis-python/src/main/resources/python/molgenis.py

```diff
@@ -0,0 +1,72 @@
+import requests;
+import json;
+import urllib;
+
+class Connection():
+	def __init__(self, url):
+	    self.url = url
+
+	def login(self, username, password):
+		response = requests.post(self.url + "v1/login", 
+			data=json.dumps({"username":username, 'password':password}), 
+			headers={"Content-Type":"application/json"})
+		if response.status_code == 200:
+			self.token = response.json()["token"]
+		return response
+
+	def _getTokenHeader(self):
```

Comment:
According to PEP 0008 (https://www.python.org/dev/peps/pep-0008/) function names should be lower case separated (e.g. _get_token_header)